src/lex.js

```diff
@@ -577,7 +577,7 @@ Lexer.prototype = {
       "super", "return", "typeof", "delete",
       "switch", "export", "import", "default",
       "finally", "extends", "function", "continue",
-      "debugger", "instanceof", "true", "false", "null"
+      "debugger", "instanceof", "true", "false", "async", "await"
```

Comment:
Did you intend to remove "null"?

Reply:
I did not. I've pushed up a "fixup" commit to address this. Strange that it didn't cause regressions; I'll look into that later.